### PR TITLE
Add feed from clipboard

### PR DESCRIPTION
**Pull request by agazso**
When adding new feed, FeedInfo finds the content of the clipboard for a link and add automatically

### CODE REVIEW

**Comment by agazso** [comment]
I also changed the timeout of fetching a Swarm feed, because it often when adding a feed.

File: src/components/FeedInfo.tsx [hunk 1]
```diff
@@ -192,9 +197,24 @@ export class FeedInfo extends React.Component<DispatchProps & StateProps, EditFe
...
+    private tryToAddFeedFromClipboard = () => {
+        const isExistingFeed = this.props.feed.feedUrl.length > 0;
+        if (!isExistingFeed) {
+            Clipboard.getString().then(value => {
+                const link = Utils.getLinkFromText(value);
+                if (link != null) {
+                    this.setState({
+                        url: link,
+                    });
+                    this.fetchFeed().then();
```
Comment (vujevits) [inline src/components/FeedInfo.tsx]: we could use await and mark tryToAddFeedFromClipboard and componentDidMount async

Comment (agazso) [reply src/components/FeedInfo.tsx]: 👍

**Comment by agazso** [comment]
Changed the code to show the QR camera only after the clipboard was checked for links and no link were found. This avoids the confusion for first time users, who get a link from someone and then when trying to add the camera permissions dialog comes up without using the camera.

Also added some explaining text to describe what's going on (the wording could be improved here, especially if we find a better name for feeds).

**Comment by vujevits** [comment]
At least with password managers I know it's a practice to flush the clipboard, I think we can argue that a feed url is sensitive data too.

**Comment by agazso** [comment]
I added clearing the clipboard after the feed was successfully fetched. I think it works better this way and it's easy to disable if we change our minds later.

**Pull request merged**